src/Components/ProjectForm.js

```diff
@@ -129,18 +163,22 @@ function ProjectForm() {
               <label>Name</label>
                 <TextField
                   data-testid="form_name"
-                  id="name"
                   label=""
                   multiline
                   maxRows={1}
                   inputProps={{ "data-testid": "form-inputName" }}
                   value={name}
                   onChange={handleNameChange}
                   fullWidth
+                  id="filled-basic"
+                  variant="filled"
+                  style={{ color:'#218888'}}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8488139)
```suggestion
                  style={{ color:"#218888"}}
```